packages/frontend/src/utils/orderByTvl.ts

```diff
@@ -9,10 +13,25 @@ export function orderByTvl<
   const archived = projects.filter((project) => project.isArchived)
   const upcoming = projects.filter((project) => project.isUpcoming)
 
-  const getTvl = (project: T) =>
-    tvlApiResponse.projects[project.id.toString()]?.charts.hourly.data.at(
-      -1,
-    )?.[1] ?? 0
+  const getTvl = (project: T) => {
```

Comment:
Will this work with the sorting on the table?

Reply:
nope but I thought it should not